assets/app/scripts/services/data.js

```diff
@@ -1019,8 +1066,15 @@ angular.module('openshiftConsole')
       namespace: namespace,
       q: params
     };
+    // TODO (bpeterse): The URL_OBJECT_SUBRESOURCE & URL_NAMESPACED_OBJECT_SUBRESOURCE
+    // have both been removed.  Need to figure out what to use now in place of these.
     if (isWebsocket) {
-      template = namespaceInPath ? URL_NAMESPACED_WATCH_LIST : URL_WATCH_LIST;
+      if (subresource) {
+        templateOptions.subresource = subresource;
+        template = namespaceInPath ? URL_NAMESPACED_OBJECT : URL_OBJECT;
+      } else {
+        template = namespaceInPath ? URL_NAMESPACED_WATCH_LIST : URL_WATCH_LIST;
+      }
```

Comment:
@liggitt and I just discussed this change.  We can now initiate a watch by passing ?watch=true, so we should stop using isWebsocket to change the path to watch/whatever, so basically this if(isWebsocket) block should be gone.  Instead anywhere else we were already calling this function with isWebsocket = true just needs to add watch:true to params being passed in.  so like from _startWatchOp